.github/workflows/build.yml

```
@@ -43,7 +43,7 @@ jobs:
         with:
           submodules: recursive
       - name: 'Build'
-        run: pip install . -v --no-deps
+        run: pip install . -v
```

Removing `--no-deps` does better job at testing the build, but it might also rebuild Flash Attention.
```suggestion
        run: pip install . -v --no-deps
```

---

The base container should already have a compatible FA installed